src/api/database/bulk-api.rst

```diff
@@ -332,6 +332,9 @@ Sending multiple queries to a database
 
     :param db: Database name
 
+    :query number page_size: Specify the number of queries in the result.
```

Comment:
For consistency with the other descriptions, I wouldn't make this a complete sentence.

```suggestion
    :query number page_size: The number of queries in the result.
```